styles/Canonical/016-No-inline-comments.yml

```
@@ -4,9 +4,7 @@ scope: raw
 nonword: true
 level: warning
 tokens:
-  - ^```.*(\n(?s:.)*#.*)+.*\n```
-  - ^```.*(\n(?s:.)*//.*)+.*\n```
+  - ^```.*\n([^\n][^\n#]*\n)*([^\n#]+#.*\n)+([^\n#]*\n)*```
+  - ^```.*\n([^\n][^\n]*\n)*([^\n\/][^\n\/]+\/\/.*\n)+([^\n][^\n]*\n)*```
```

```suggestion
  - ^```.*\n([^\n`][^\n]*\n|\n)*([^\n\/][^\n\/]+\/\/.*\n)+([^\n`]*\n)*```
```
Avoids looking at more than a single instance of a code block. Lines can't contain code block delimiters, also allows for a blank line in the code before the identified line with the inline comment.